src/algebra/big_operators/fin.lean

```diff
@@ -139,6 +142,40 @@ by simpa only [prod_univ_add, fintype.prod_eq_one _ hf, mul_one]
 
 end fin
 
+/-- Equivalence between `fin n → fin m` and `fin (m ^ n)`. -/
+def fin_function_fin_equiv {m n : ℕ} : (fin n → fin m) ≃ fin (m ^ n) :=
```

Comment:
```suggestion
@[simps]
def fin_function_fin_equiv {m n : ℕ} : (fin n → fin m) ≃ fin (m ^ n) :=
```
but check it produces something reasonable.

Reply:
Yup, the autogenerated lemmas look fine.